src/bundle/Resources/views/themes/admin/ui/component/macros.html.twig

```diff
@@ -8,3 +8,18 @@
         {%- endif -%}
     {%- endfor -%}
 {%- endmacro -%}
+
+{% macro results_headline(count, has_filters = false, phrase = null) %}
```

Comment:
Since we want have everywhere the same list header on pages, there was implemented macro which can be used in every package. It takes filters and phrase if search or filters are avaiable.